docs/ApiGuidelines.md

```diff
@@ -46,8 +55,10 @@ Experimental API requirements:
 * Experimental APIs must be in a header file that is published to the SDK output directory. They
   can be in their own header, or in a header file that is shared with public APIs but only
   if surrounded by an appropriate ifdef (e.g., "#ifdef OE\_CONTEXT\_SWITCHLESS\_EXPERIMENTAL\_FEATURE").
```

Comment:
Nit: consider using code markdown for readability and not having to use character escapes (e.g. `#ifdef OE_CONTEXT_SWITCHLESS_EXPERIMENTAL_FEATURE`)